src/sentry/integrations/jira/configure.py

```diff
@@ -55,12 +55,18 @@ def handle(self, request):
                 'login_url': absolute_uri(reverse('sentry-login')),
             })
 
-        organizations = request.user.get_orgs().filter(
+        organizations = list(request.user.get_orgs().filter(
             id__in=OrganizationMember.objects.filter(
                 role__in=[r.id for r in roles.get_all() if r.is_global],
                 user=request.user,
             ).values('organization'),
-        )
+        ))
+
+        # TODO(jess): remove after wide release
+        organizations = [
+            o for o in organizations if features.has(
+                'organizations:jira-integration', o, actor=request.user)
+        ]
```

Comment:
Do you think we really need this? Since the whole frontend is mostly flagged out

Reply:
this is to handle people coming from the atlassian marketplace. otherwise people who go through that flow could install, link to an org and then just not be able to see the integration they've just configured in their list. seems better to prevent them from linking/tell them to enable EA if they want to link